tests/unit/moduleapi/moduleconfigs.tcl

```diff
@@ -243,5 +245,14 @@ start_server {tags {"modules"}} {
             assert_equal [r config get moduleconfigs.memory_numeric] "moduleconfigs.memory_numeric 1024"
         }
     }
-}
+    test {Module Update Args } {
+       r module load $testmoduleparameter 10 20 30
 
+       set t [r module list]
+       set modulename [lmap x [r module list] {dict get $x name}]
+       assert_not_equal [lsearch $modulename moduleparameter] -1
+       assert_equal "{10 20 30}" [lmap x [r module list] {dict get $x args}]
+       r testmoduleparameter.update.parameter 40 50 60 70
```

Comment:
It should return OK on success, right?

```suggestion
       assert_equal OK [r testmoduleparameter.update.parameter 40 50 60 70]
```

Reply:
Or maybe it returns the string "Module runtime args test", the reply sent by this command in the previous commits.

```suggestion
       assert_equal "Module runtime args test" [r testmoduleparameter.update.parameter 40 50 60 70]
```

... but OK seems more normal I think :grin:

Reply:
I update this part as assert_equal OK [r testmoduleparameter.update.parameter 40 50 60 70]